fn.go

```diff
@@ -131,8 +232,14 @@ func (f *Function) RunFunction(_ context.Context, req *fnv1beta1.RunFunctionRequ
 		}
 
 		unstructuredData := composed.Unstructured{}
-		unstructuredDataByte, _ := json.Marshal(jobTemplate)
-		json.Unmarshal(unstructuredDataByte, &unstructuredData)
+		unstructuredDataByte, err := json.Marshal(jobTemplate)
+		if err != nil {
+			fmt.Println(err)
+		}
+		err = json.Unmarshal(unstructuredDataByte, &unstructuredData)
+		if err != nil {
+			fmt.Println(err)
+		}
```

Comment:
@ca7alindev Job not required anymore, you could send final message right from this function, or possible i lost what this job doing?

Reply:
@evghen7 This job is related to issue #5, there i will replace it with sending results logic